tests/test_benchmark_free_energy.py

```diff
@@ -80,19 +80,22 @@ def setup_hif2a_single_topology_leg(host_name: str, n_windows: int, lambda_endpo
     host = setup_optimized_host(single_topology, host_config) if host_config else None
 
     # Use the bisection schedule to require the least amount of minimization during bisection
```

Comment:
Should this comment be removed?

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1355/commits/76a6dedb9ed459c39ede0f1a5e0e6822f45a9478